### DISCUSSION TITLE
meta keywords

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1269] [Q&A]
It's easy enough to add the keywords meta tag to the page head, but I'm wondering why it wasn't included as a standard setting like Description is.

**Comment by meetmandeep** [comment]
Meta Keywords are no longer relevant in world of SEO and hence they were removed from Vanjaro. It's extremely important to us that Vanjaro has a clean and relevant UI only.
